src/scss/patterns/_sizes.scss

```diff
@@ -1,28 +1,7 @@
-.size-full {
-  width: 100%;
-}
-
-.size-half {
-  width: 50%;
-}
-
 .size-quarter {
   width: 25%;
 }
 
-// Extend thing outside its container on one side only
-.extend-left {
-  --column: wide / page;
-
-  width: 100%;
-}
-
-.extend-right {
-  --column: page / wide;
-
-  width: 100%;
-}
-
 // Extend thing past the width of its container on both sides
 .extend-small {
```

Comment:
This one isn't documented?

Reply:
See my case above that it may still be useful in other situations, like in nunjucks. If it's a thing we do sometimes, I think it makes sense to have a standard way of doing it.